server/schedulers/hot_region.go

```diff
@@ -776,7 +789,10 @@ func (bs *balanceSolver) pickDstStores(filters []filter.Filter, candidates []*co
 			detail := bs.stLoadDetail[store.GetID()]
 			maxLoads := detail.LoadPred.max().Loads
 			if slice.AllOf(maxLoads, func(i int) bool {
-				return maxLoads[i]*dstToleranceRatio < detail.LoadPred.Expect.Loads[i]
+				if statistics.IsSelectedDim(i) {
+					return maxLoads[i]*dstToleranceRatio < detail.LoadPred.Expect.Loads[i]
+				}
+				return true
```

Comment:
It seems the `qps` won't be calculated in hot scheduling for now?

Reply:
yes